samples/client/petstore/go/go-petstore/enum_class.go

```diff
@@ -10,5 +10,12 @@
 
 package petstore
 
-type EnumClass struct {
-}
+type EnumClass string
+
+// List of EnumClasss
+const (
+	_abc EnumClass = "_abc"
```

Comment:
Sorry I'm not exactly sure how to use those functions in the moustache template. Could you shed some light about how to use them?